controllers/managedocs_controller.go

```diff
@@ -1058,6 +1079,95 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcilePrometheusRemoteWrite() (*promv1.RemoteWriteSpec, error) {
+	rhobsSecret := &corev1.Secret{}
+	rhobsSecret.Name = prometheusRemoteWriteSecretName
+	rhobsSecret.Namespace = r.namespace
```

Comment:
- maybe this initilization should go into `initReconciler`
- we can access `DeploymentType` in there as well

Reply:
I don;t think we really need it anywhere else. This variable should be fine locally.
@nb-ohad let me know if you think otherwise

Reply:
Changed implementation